pyuvdata/tests/test_utils.py

```diff
@@ -955,6 +955,8 @@ def test_uvcalibrate_apply_gains():
     # read data
     uvd = pyuvdata.UVData()
     uvd.read(os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAA'))
+    # give it an x_orientation
+    uvd.x_orientation = 'e'
```

Comment:
It doesn't look like this tests that the x_orientation is actually used properly.

Reply:
I think it does because uvcalibrate fails otherwise. I don't think the codecov report is unrelated, because the "misses" are in version.py and uvbase.py, which aren't affected by this PR.

Reply:
Ah, ok. Then you should tick the box above that says you added a test that fails as a result of the bug. Can you also rebase?